sdcm/results_analyze/__init__.py

```diff
@@ -117,48 +167,26 @@ def __init__(self, es_index, es_doc_type, send_email, email_recipients, logger=N
         )
 
     @staticmethod
-    def _get_setup_details(test_doc, is_gce):
-        setup_details = {'cluster_backend': test_doc['_source']['setup_details'].get('cluster_backend')}
-        if setup_details['cluster_backend'] == "aws":
-            setup_details['ami_id_db_scylla'] = test_doc['_source']['setup_details']['ami_id_db_scylla']
-        for setup_param in QueryFilter(test_doc, is_gce).setup_instance_params():
-            setup_details.update(
-                [(setup_param.replace('gce_', ''), test_doc['_source']['setup_details'].get(setup_param))])
-        setup_details.update(test_doc['_source']['versions']['scylla-server'])
-        return setup_details
+    def _get_result_details_from_es(test_doc, is_gce):
+        result_details = dict()
 
-    @staticmethod
-    def _get_grafana_snapshot(test_doc):
-        grafana_snapshots = test_doc['_source']['test_details'].get('grafana_snapshots')
-        if grafana_snapshots and isinstance(grafana_snapshots, list):
-            return grafana_snapshots
-        elif grafana_snapshots and isinstance(grafana_snapshots, str):
-            return [grafana_snapshots]
-        else:
-            return []
-
-    @staticmethod
-    def _get_grafana_screenshot(test_doc):
-        grafana_screenshots = test_doc['_source']['test_details'].get('grafana_screenshot')
-        if not grafana_screenshots:
-            grafana_screenshots = test_doc['_source']['test_details'].get('grafana_screenshots')
-
-        if grafana_screenshots and isinstance(grafana_screenshots, list):
-            return grafana_screenshots
-        elif grafana_screenshots and isinstance(grafana_screenshots, str):
-            return [grafana_screenshots]
-        else:
-            return []
+        return result_details
```

Comment:
@fgelcer this method should return empty dict?

Reply:
ops, it was a leftover from previous code, and it has moved to other functions, hence it was now removed.